nengo/dists.py

```diff
@@ -99,9 +102,9 @@ class Uniform(Distribution):
         If true, sample from a uniform distribution of integers. In this case,
         low and high should be integers.
     """
-    low = NumberParam()
-    high = NumberParam()
-    integer = BoolParam()
+    low = NumberParam('low')
```

Comment:
Or I guess another option would be to just throw an error if they pass a parameter to `config.set_param` that doesn't have a label set.